internal/pkg/skuba/deployments/ssh/ssh.go

```diff
@@ -165,7 +165,7 @@ func (t *Target) internalSshWithStdin(silent bool, stdin string, command string,
 		finalCommand = fmt.Sprintf("sudo sh -c '%s'", finalCommand)
 	}
 	if !silent {
-		klog.V(1).Infof("running command: %q", finalCommand)
+		klog.V(2).Infof("running command: %q", finalCommand)
```

Comment:
Actually, these log messages in ssh.go make me nervous, as I don't see any sort of security check in place that would prevent any sensitive information used in the ssh command from being logged.  I am hoping others with more experience in skuba can comment.

Reply:
there is a "silent" ssh. this is the one we use for secure stuff that does not log anything.

Reply:
Good point, and good to know where we place the trust that the command is not run incorrectly.